### ISSUE TITLE
Pushover alarms don't respect snooze duration

### ISSUE THREAD

**Issue by jimrandomh** [nightscout/cgm-remote-monitor#3487]
When I have a high BG, Nightscout sends a pushover alert, with "urgent" priority, every fifteen minutes. This makes the Pushover alerts channel so noisy that genuinely-urgent alerts like OpenAPS' carbs-required alert are hard to notice. I have set ALARM_HIGH_MINS and ALARM_URGENT_HIGH_MINS to larger numbers, with no effect. Nor does it respect the BG_HIGH setting; the text in the Pushover notification changes between "Warning" and "Urgent" based on whether it's greater than BG_HIGH, but the actual Pushover priority is set to "Emergency" no matter what.

**Comment by jimrandomh** [comment]
My overall setup is:

    Intel Edison running Jubilinux and OpenAPS 0.6
    Dexcom G4 connected by USB OTG cable to the Edison
    Nightscout running on Heroku, fork https://github.com/jimrandomh/cgm-remote-monitor

Nightscout's config is:

    ALARM_HIGH on
    ALARM_HIGH_MINS 30 60 90
    ALARM_LOW on
    ALARM_TIMEAGO_URGENT on
    ALARM_TIMEAGO_URGENT_MINS 30
    ALARM_TIMEAGO_WARN on
    ALARM_TIMEAGO_WARN_MINS 30
    ALARM_TYPES predict
    ALARM_URGENT_HIGH on
    ALARM_URGENT_HIGH_MINS 40
    ALARM_URGENT_LOW on
    API_SECRET <...>
    AUTH_DEFAULT_ROLES status-only
    BG_HIGH 260
    BG_LOW 60
    BG_TARGET_BOTTOM 80
    BG_TARGET_TOP 180
    BRIDGE_PASSWORD
    BRIDGE_USER_NAME
    CUSTOM_TITLE James Babcock BG
    DEVICESTATUS_ADVANCED true
    DISABLE treatmentnotify
    DISPLAY_UNITS mg/dl
    ENABLE openaps pump iob basal careportal sage cage rawbg cob
    MAKER_ANNOUNCEMENT_KEY 
    MAKER_KEY 
    MONGO_COLLECTION entries
    MONGODB_URI mongodb://<...>.mlab.com:/<...>
    NIGHT_MODE off
    OPENAPS_ENABLE_ALERTS true
    OPENAPS_FIELDS status-symbol status-label iob meal-assist rssi
    OPENAPS_RETRO_FIELDS status-symbol status-label iob meal-assist rssi
    PAPERTRAIL_API_TOKEN <...>
    PUMP_FIELDS battery reservoir clock status
    PUMP_URGENT_BATT_V 1.3
    PUSHOVER_ANNOUNCEMENT_KEY
    PUSHOVER_API_TOKEN <...>
    PUSHOVER_USER_KEY <...>
    SHOW_RAWBG never
    THEME default
    TIME_FORMAT 24

**Comment by jimrandomh** [comment]
I can't remove Nightscout's ability to send Pushover notifications entirely, because the "not looping" notification works, and I need it. But I don't need high-BG notifications from Nightscout, because my CGM is already providing them, and Nightscout's version is spammy. I've tried changing ALARM_HIGH to "off"; that didn't do anything. I changed BG_TARGET_TOP to a huge number; that changes the emergency-priority pushover "Urgent, HIGH" notification into an emergency-priority pushover "Warning, Check BG predicted" notification. I think I'm going to go comment out some big swaths of Javascript.
